### ISSUE TITLE
Use HTTPS Only

### ISSUE THREAD

**Issue by mjcarroll** [urbanengine/OpenHuntsville#123]
If cert cost is an issue, look into using letsencrypt.

**Comment by davidhjones** [comment]
Look into this https://github.com/substrakt/letsencrypt-heroku

**Comment by RandomlyKnighted** [comment]
The certificate has been uploaded. All we need to do now is update some of our DNS records and we should be good to go. I ended up following this [blog post](https://medium.com/@bantic/free-tls-with-letsencrypt-and-heroku-in-5-minutes-807361cca5d3) to set it all up.

I had to make one change in order for it to work. The blog post says to run this command:

`sudo heroku certs:add /etc/letsencrypt/live/domain/fullchain.pem /etc/letsencrypt/live/domain/privkey.pem` 

However, the file path was a little different and I also had to add `--app openhsv` to the end of the command. Here's the command I ended up running to get the Heroku CLI to upload the certificate:

`sudo heroku certs:add /etc/letsencrypt/live/openhsv.com/fullchain.pem /etc/letsencrypt/keys/0000_key-certbot.pem --app openhsv`
